index.js

```diff
@@ -19,7 +19,20 @@ app.use(express.static(path.join(__dirname, 'public'))); // Serving static files
 app.use(bodyParser.json()); // Middleware to parse JSON request bodies
 
 // CORS configuration
+const allowedOrigins = ['http://localhost:3000', 'mongodb+srv://yukiontour:Test123@cluster0.vimspko.mongodb.net/?retryWrites=true&w=majority&appName=Cluster0'];
```

Comment:
Hi @YukiOnTour ,
`mongodb+srv://yukiontour:Test123@cluster0.vimspko.mongodb.net/?retryWrites=true&w=majority&appName=Cluster0` NOT an example of an allowed origin. That's your mongodb connection URI. The allowedOrigin should be domains like 'http://localhost:1234', since your frontend on local runs on that port. The other allowedOrigin should be the link to the deployed myFix-client application.